### DISCUSSION TITLE
Possibility to share birdnet-pi data after recording without internet

### DISCUSSION THREAD

**Discussion by beppefiocco** [mcguirepr89/BirdNET-Pi#815] [General]
I use Birdnet-Pi without internet, then i want to share the recorded data, there's a possibility to clone the site on the pi and upload to a domain?
Or at least a sort of summary of the recordings?

Thank you for the beautiful project!

**Comment by ehpersonal38** [comment]
You mean like you have a remote BirdNET-Pi that you periodically collect the data from and now want to display this data on a separate site?

That's very doable, you'd have to build a website that compiled the data from `birds.db` and displayed it in whatever format you wanted. You could probably look up how to install Streamlit (the service behind the BirdNET-Pi Species Stats page) on your server and then steal our .py file that generates the stats: https://github.com/mcguirepr89/BirdNET-Pi/blob/main/scripts/plotly_streamlit.py
